e2e/testcases/helm_sync_test.go

```diff
@@ -237,7 +238,7 @@ func TestHelmARTokenAuth(t *testing.T) {
 	nt := nomostest.New(t,
 		ntopts.SkipMonoRepo,
 		ntopts.Unstructured,
-		ntopts.RequireGKE(t),
+		//ntopts.RequireGKE(t),
```

Comment:
Is this test required to run on GKE cluster? If not, remove the comment.

Reply:
done.